packages/react/src/components/GroupedList/GroupedListV2.test.tsx

```diff
@@ -0,0 +1,350 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import { SelectionMode, Selection } from '../../Selection';
+import { GroupedListV2_unstable as GroupedListV2 } from './GroupedListV2';
+import { DetailsRow } from '../DetailsList/DetailsRow';
+import { List } from '../../List';
+import { GroupShowAll } from './GroupShowAll';
+import { Link } from '../../Link';
+import { GroupHeader } from './GroupHeader';
+import { getTheme } from '../../Styling';
+import * as path from 'path';
+import { isConformant } from '../../common/isConformant';
+import type { IGroup } from './GroupedList.types';
+import type { IColumn } from '../DetailsList/DetailsList.types';
+
+describe('GroupedListV2', () => {
+  isConformant({
+    Component: GroupedListV2,
+    displayName: 'GroupedListV2_unstable',
+    componentPath: path.join(__dirname, 'GroupedListV2.ts'),
+    requiredProps: {
+      items: [],
+      onRenderCell: () => {
+        return <div />;
+      },
+    },
+    // Problem: Ref is not supported
+    // Solution: Convert to FunctionComponent and support using forwardRef
```

Comment:
Should this be done, to switch to forwardRef?

Reply:
I'm not sure. Technically GroupedListV2 is still a class component which was done for API compatibility as GroupedList exposes public API methods on the class. I supposed `useImperativeHandle` can be used to replace those but I'm not aware of any benefits aside from enabling this conformance test.

Think it's worth the effort to change it?